src/libaktualizr/uptane/secondaryconfig.h

```diff
@@ -16,8 +16,6 @@ enum class SecondaryType {
 
   kVirtual,  // Virtual secondary (in-process fake implementation).
 
-  kLegacy,  // Deprecated. Do not use.
-
```

Comment:
Can we get rid of this whole `SecondaryType` concept? The type should be opaque to libaktualizr, right? This probably depends on getting virtual secondaries and the rest out of libaktualizr, though.

Reply:
Yes, you're assumption is correct, it depends on the virtual secondaries removal out of libaktualizr. I am planning to do it in scope of OTA-2488.